diff --git a/util/quake.lua b/util/quake.lua
index 4c42803..9bcb81e 100644
--- a/util/quake.lua
+++ b/util/quake.lua
@@ -123,7 +123,7 @@ function quake:new(config)
 
     conf.app        = conf.app       or "xterm"    -- application to spawn
     conf.name       = conf.name      or "QuakeDD"  -- window name
-    conf.argname    = conf.argname   or "-name %s" -- how to specify window name
+    conf.argname    = conf.argname   or "--classname %s" -- how to specify window name
     conf.extra      = conf.extra     or ""         -- extra arguments
     conf.border     = conf.border    or 1          -- client border width
     conf.visible    = conf.visible   or false      -- initially not visible
